configure.ac

```diff
@@ -301,6 +301,7 @@ AC_CHECK_FUNCS( futimes )
 AC_CHECK_FUNCS( wcslcpy lrand48_r killpg )
 AC_CHECK_FUNCS( backtrace_symbols getifaddrs )
 AC_CHECK_FUNCS( futimens clock_gettime )
+AC_CHECK_FUNCS( localtime_r )
```

Comment:
It looks like it's going to be mingw that doesn't have this - seems like it can be expected on cygwin? Perhaps it's not an issue?